.github/workflows/comment-on-stale-external-pr.yml

```diff
@@ -0,0 +1,38 @@
+name: Comment on Stale External PRs
+on:
+  schedule:
+    # runs at minute 0 of the 6th and 18th hour every day.
+    - cron:  '0 6,18 * * *'
+
+jobs:
+  comment_on_stale_external_prs:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content'
```

Comment:
why do we need to specify repo ?

Reply:
We don't want these workflows to get triggered on forks of the content repo. The runs would fail on forks creating lots of noise for the owner of the fork.